pkg/controller/elasticsearch/driver/upgrade_test.go

```diff
@@ -36,11 +36,11 @@ func Test_podsToUpgrade(t *testing.T) {
 			args: args{
 				statefulSets: sset.StatefulSetList{
 					sset.TestSset{
-						Name: "masters", Replicas: 2, Master: true,
+						Name: "masters", Namespace: TestEsNamespace, Replicas: 2, Master: true,
```

Comment:
I'm surprised to see that in the PR.
Does it fail if not set?

Reply:
Yeah, it was surprising to me that this ever passed. The NS was added in 
9d3aaa8 so if you check out any of the ones before that you should be able to see.